### ISSUE TITLE
Rollup not recognized as a command when running grunt

### ISSUE THREAD

**Issue by nheminger** [Esri/ago-assistant#176]
Running a grunt --force on Windows results in

```
Running "shell:command" (shell) task
'rollup' is not recognized as an internal or external command,
operable program or batch file.
Warning: Command failed: rollup -c
'rollup' is not recognized as an internal or external command,
operable program or batch file.
 Used --force, continuing.
```
Could it be a dependency issue?

**Comment by nheminger** [comment]
Running `npm install -g rollup` fixes the problem. Possible to fix the dependencies or add documentation to the readme instructions so those running grunt will have it work?

**Comment by jgravois** [comment]
the dependency is already listed in the [`package.json`](https://github.com/Esri/ago-assistant/blob/master/package.json#L43). 
are you sure you ran a fresh `npm install` since pulling down the latest?

**Comment by nheminger** [comment]
Yes, deleted the node_modules directory and ran a fresh `npm install`. Running npm 4.5.0 if that matters.

**Comment by jgravois** [comment]
interesting.

do you see any errors during install? 
is a directory called rollup not created?

**Comment by nheminger** [comment]
Will try a fresh checkout and will verify if a directory rollup gets created. Didn't see any errors.

Does the -g install flag make it available via the shell command whereas the dependency prevents that?

**Comment by ecaldwell** [comment]
This was also a complicating factor in #174. It seems that shell command only works with a globally installed rollup. I don't think there's any way to have `npm install` add it globally from `package.json`. Is there an alternative way of writing the shell command to reference the local path to rollup?

**Comment by nheminger** [comment]
@ecaldwell Sounds like that is the issue.

Excellent, maybe there is a way the shell command can be reworked - perhaps direct grunt task invocation of rollup such as this I found: [grunt-rollup](https://www.npmjs.com/package/grunt-rollup)?

If not, maybe a FAQ or addition to the README so people know to do the global install.

**Comment by ecaldwell** [comment]
I think the FAQ/README should be updated first, then we can explore alternatives. I actually had tried using grunt-rollup originally, but couldn't get it to work. Maybe a newer version of it works.

**Comment by ecaldwell** [comment]
I think the solution is to specify the path to `node_modules` in the shell command. You can see this in my Gruntfile on #178.

Now the command looks like this `rollup: "<%= node_modules %>rollup -c"` where `node_modules` is defined earlier in the config as `.\\node_modules\\.bin\\`.

I tried a variety of combinations to get this to execute properly and this was the only way that worked. Unfortunately, the escaped backslashes are necessary—using forward slashes caused errors. It would be good for someone on a Unix machine to verify if this works.

**Comment by ecaldwell** [comment]
Resolved in #201.
